practices/practice-two/src/pages/index.html

```diff
@@ -0,0 +1,154 @@
+                  </li>
+                  <li class="c-md-navigations-item">
+                    <a
+                      href=""
+                      class="c-md-navigations-item-link"
+                      >about us</a
+                    >
+                  </li>
+                  <li class="c-md-navigations-item">
+                    <a
+                      href=""
+                      class="c-md-navigations-item-link"
+                      >pricing</a
+                    >
+                  </li>
+                  <li class="c-md-navigations-item">
+                    <a
+                      href=""
+                      class="c-md-navigations-item-link"
+                      >contact us</a
+                    >
+                  </li>
+                  <li class="c-md-navigations-item">
+                    <button
+                      class="btn btn-secondary c-md-navigations-item-btn"
+                    >
+                      get started
+                    </button>
+                  </li>
+                </ul>
+              </div>
+              <ul class="c-xl-navigations">
+                <li class="c-xl-navigations-item">
+                  <a
+                    href=""
+                    class="c-xl-navigations-item-link active"
+                    >home</a
+                  >
+                </li>
+                <li class="c-xl-navigations-item">
+                  <a
+                    href=""
+                    class="c-xl-navigations-item-link"
+                    >about us</a
+                  >
+                </li>
+                <li class="c-xl-navigations-item">
+                  <a
+                    href=""
+                    class="c-xl-navigations-item-link"
+                    >pricing</a
+                  >
+                </li>
+                <li class="c-xl-navigations-item">
+                  <a
+                    href=""
+                    class="c-xl-navigations-item-link"
+                    >contact us</a
+                  >
+                </li>
+                <li class="c-xl-navigations-item">
+                  <button
+                    class="btn btn-secondary c-xl-navigations-item-btn"
+                  >
+                    get started
+                  </button>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="header-introduce-wrapper">
+            <div class="header-introduce-item">
+              <h1 class="header-introduce-item-title">
+                Welcome to the Wapik
+              </h1>
+              <p class="header-introduce-item-description">
+                Wapik is multipurpose template, with modern
+                and smart design. Wapik is perfect template
+                for you!
+              </p>
+              <button
+                class="btn btn-primary header-introduce-item-btn"
+              >
+                LEARN MORE
```

Comment:
Use styles to make uppercase

Reply:
i use uppercase in css file

Reply:
So why LEARN MORE? Learn more instead of